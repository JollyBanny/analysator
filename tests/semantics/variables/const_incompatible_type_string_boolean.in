const a : string = true;

begin
end.