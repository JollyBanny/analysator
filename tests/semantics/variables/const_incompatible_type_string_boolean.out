[Line: 1; Column: 11]	incompatible types: got 'boolean' expected 'string'
